admin/resources/js/components/menu/SideMenu.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactElement } from "react";
+
+export const SideMenu: React.FunctionComponent<{
+  items: ReactElement[];
+}> = ({ items }) => {
```

Comment:
I like this. Adding the type directly over creating a type/interface. Especially if there's less then 3 fields.

Reply:
Yeah, I've started doing that for small interfaces which aren't used anywhere else.